tests/autoyast/verify_system_lvm.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Basic verification of system after autoyast installation
+# with lvm on btrfs.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent 'installbasetest';
```

Comment:
We should use appropriate base class as it's not installation module, namely `y2_module_consoletest.pm`.

Reply:
ok